### PR TITLE
Add Pacbio views

### PR DESCRIPTION
**Pull request by islean**
## Description

Closes https://github.com/Clinical-Genomics/add-new-tech/issues/119.

### Added

- View for PacbioSampleRunMetrics
- View for PacbioSequencingRun

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/server/admin.py [hunk 1]
```diff
@@ -666,3 +676,82 @@ class IlluminaSampleSequencingMetricsView(BaseView):
...
+        "p2_percent",
+        "polymerase_mean_length",
+        "polymerase_read_length_n50",
+        "polymerase_mean_longest_subread",
+        "polymerase_mean_longest_subread_n50",
+        "control_reads",
+        "control_mean_read_length",
+        "control_mean_read_concordance",
+        "control_mode_read_concordance",
+        "productive_zmvs",
```
[the remark this answers is not in the document]
Comment (diitaz93) [reply cg/server/admin.py:706]: To clarify, I mean `productive_zmvs`

File: cg/server/admin.py [hunk 2]
```diff
@@ -666,3 +676,82 @@ class IlluminaSampleSequencingMetricsView(BaseView):
...
+
+
+class PacbioSampleRunMetricsView(BaseView):
+    column_list = [
+        "smrt_cell",
+        "sample",
+        "hifi_reads",
+        "hifi_mean_read_length",
+        "hifi_median_read_quality",
+        "polymerase_mean_read_length",
```
[the remark this answers is not in the document]
Comment (islean) [reply cg/server/admin.py:751]: What was the point of persisting them in the recent PR then? We removed some columns and only added this one.

[the remark this answers is not in the document]
Comment (islean) [reply cg/server/admin.py:751]: I had some recollection that we wanted to keep this one but remove the others?

File: cg/server/admin.py [hunk 3]
```diff
@@ -666,3 +676,82 @@ class IlluminaSampleSequencingMetricsView(BaseView):
...
+    ]
+    column_formatters = {
+        "smrt_cell": PacbioSmrtCellView.view_smrt_cell_link,
+        "sample": SampleView.view_sample_link,
+    }
+    column_searchable_list = ["sample.internal_id", "instrument_run.device.internal_id"]
```
[the remark this answers is not in the document]
Comment (islean) [reply cg/server/admin.py:757]: Already checked 😎

File: cg/server/admin.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (diitaz93) [reply cg/server/admin.py:742]: This info may be useful for Jesper and Felix, but not for Production (that I am aware of), that is why I think we keep it in the database but not show it on the admin view

[the remark this answers is not in the document]
Comment (diitaz93) [reply cg/server/admin.py:742]: we can always add it later if production requests it

**Comment by islean** [comment]
Deployed to cg-services-stage and cg-services-prod

**Pull request merged**